test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -606,6 +614,8 @@ public void startup(ICluster cluster)
                 // see org.apache.cassandra.service.CassandraDaemon.setup
                 StorageService.instance.populateTokenMetadata();
 
+                logSystemInfo();
```

Comment:
nitL Just to try to keep maximum continuity w/ `CassandraDaemon`, I'd move this just before `CommitLog.instance.start();` above.